lib/__tests__/init.test.ts

```diff
@@ -100,4 +106,30 @@ describe("init", () => {
       "https://insights.de.algolia.io"
     );
   });
+  it("should set userToken to ANONYMOUS if environment supports cookies", () => {
+    const supportsCookies = jest
+      .spyOn(utils, "supportsCookies")
+      .mockReturnValue(true);
+    const setUserToken = jest.spyOn(AlgoliaInsights, "setUserToken");
+
+    AlgoliaInsights.init({ apiKey: "***", appId: "XXX", region: "de" });
+    expect(setUserToken).toHaveBeenCalledWith(
```

Comment:
We should also check that it's been called only once.

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/4803db67287b9f2c98947f8dd22b7674a4b672e8